.travis.yml

```diff
@@ -50,11 +50,11 @@ script:
   - npm run lint:all
   - npm run test:ci:all
   - npm run docs:coverage
-  - if [[ $TRAVIS_TAG == "" && ${TRAVIS_EVENT_TYPE} != "cron" ]]; then npm run build:showcase:ghpages:ci-test-env; else npm run build:showcase:ghpages; fi
+  - if [[ {$TRAVIS_TAG} == "" && ${TRAVIS_EVENT_TYPE} != "cron" ]]; then npm run build:showcase:ghpages:ci-test-env; else npm run build:showcase:ghpages; fi
```

Comment:
Are the brackets in this line correct? Maybe not... Shouldn't they be `${...}` ?

Reply:
And in fact, I think it is indeed wrong because I saw that Travis run `npm run build:showcase:ghpages` instead of `npm run build:showcase:ghpages:ci-test-env`

Reply:
You're right, I made a mistake here. My bad. I'll fix that 😊